prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Brittany Boughter';
+    console.log(fullName);
+    var numberOfPets = '2';
+    console.log(numberOfPets);
+    var likesVegetables = true;
+    console.log(likesVegetables);
+    var nothing = 'null';
+    console.log(nothing);
```

Comment:
We want to leave the quotes off of `null` so that the javascript engine doesn't interpret it as a string.